addons/mail/static/src/core/common/attachment_model.js

```diff
@@ -76,6 +89,28 @@ export class Attachment extends FileModelMixin(Record) {
     get previewName() {
         return this.voice ? _t("Voice Message") : this.name || "";
     }
+
+    async setPdfThumbnail(width = 180, height = 110) {
```

Comment:
This size feels arbitrary. Shouldn't we aim a little bit higher so we can more easily tweak its style in the future? Or is that already higher?

Reply:
It's the size of the current thumbnail in the UI.
I don't know what's best

<img width="808" height="205" alt="image" src="https://github.com/user-attachments/assets/0a28ba0c-aa4e-4f19-9a4e-9d00310e58c4" />